UnicornEngine/video/source/Camera.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/Camera.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+Camera::Camera(const glm::vec3& position, const glm::vec3& direction) : m_aspect(0)
+                                                                      , m_position(position)
```

Comment:
Just to be clear - you have 70 spaces indent here. Optimal line length is historically 80 (both for code readability and code review) and you're clocking over a 100 with upVector. I have to insist on smaller indent in this case.